tests/test_utils.py

```diff
@@ -930,6 +930,51 @@ def test_read_csv(csv, result):
         pd.testing.assert_frame_equal(obj, result)
 
 
+@pytest.mark.parametrize(
+    'index, extension, expected_index',
+    [
+        (
+            audformat.filewise_index(),
+            'mp3',
+            audformat.filewise_index(),
+        ),
+        (
+            audformat.segmented_index(),
+            'mp3',
+            audformat.segmented_index(),
+        ),
+        (
+            audformat.filewise_index(['f1.wav', 'f2.wav']),
+            'mp3',
+            audformat.filewise_index(['f1.mp3', 'f2.mp3']),
+        ),
+        (
+            audformat.segmented_index(['f1.wav', 'f2.wav']),
+            'mp3',
+            audformat.segmented_index(['f1.mp3', 'f2.mp3']),
+        ),
+        (
+            audformat.filewise_index(['f1.WAV', 'f2.WAV']),
+            'MP3',
+            audformat.filewise_index(['f1.MP3', 'f2.MP3']),
+        ),
+        (
+            audformat.filewise_index(['f1', 'f2.wv']),
+            'mp3',
+            audformat.filewise_index(['f1', 'f2.mp3']),
+        ),
+        (
+            audformat.filewise_index(['f1.wav', 'f2.wav']),
+            '',
+            audformat.filewise_index(['f1.', 'f2.']),
```

Comment:
Here, I'm also not sure if we should return `['f1', 'f2']` instead?

Reply:
Yes, I think that makes more sense.

Reply:
I changed it and extended the docstring, see the description above.